includes/class-newspack-popups-api.php

```diff
@@ -409,6 +431,7 @@ public function set_utm_source_transient( $utm_source ) {
 	public function set_utm_medium_transient() {
 		$transient_name = self::get_suppression_data_transient_name( 'utm_medium' );
 		if ( $transient_name ) {
+			$this->update_cache( $transient_name, true );
 			set_transient( $transient_name, true, 0 );
```

Comment:
Tiny nitpick: Is there ever a situation where `$this->update_cache` is not followed by `set_transient`? If not, you could move `set_transient` into `update_cache` (or something like that) and save a bunch of repeating yourself.

Reply:
Good idea. https://github.com/Automattic/newspack-popups/pull/208/commits/9e902337a0d46df83e7fc6341db56baa0e347566